CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## [0.7.2] - 2023-02-17
+
+### Features
```

Comment:
#3003 requires a breaking change notice ?

Reply:
I do not think this is a breaking change: 
- user is no aware of this change
- new version is compatible with the old when upgrading

Reply:
Deploy response time will still face overflow issue from upgrading. At least a notable upgrade notice should exists in ChangeLog

Reply:
ok